ntrans_dataprep.py

```diff
@@ -77,6 +92,7 @@ def generate_ngrams_from_corpus():
     The sentences are formatted before they are split into N-grams and added to lists, depending on
     the N-gram length.
     """
+    # TODO: Update Docstring to explain csv split process
```

Comment:
Don't write internal details to docstrings. If someone cares about them, they can read the code, and they won't trust the docstring to be up-to-date anyway.

Reply:
Nevermind. I misunderstood what you meant with this comment.